cypress/e2e/tests/pages/charts/monitoring-istio.spec.ts

```diff
@@ -186,4 +189,42 @@ describe('Charts', { tags: ['@charts', '@adminUser'] }, () => {
       });
     });
   });
+
+  describe('Istio', () => {
+    const istioVersion = '103.0.0%2Bup1.18.2';
```

Comment:
Comment only. At some point we'll need to change this to navigate to the chart detail page first and install from there to get the latest available version. Monitoring above uses this process (`chartsPage.goToInstall()`)